CollAction/ValidationAttributes/DateWithinMonthsAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace CollAction.ValidationAttributes
+{
+    public class DateWithinMonthsAttribute : ValidationAttribute
+    {
+        private readonly int _months;
+
+        public String OfDateProperty { get; set; }
+
+        public DateWithinMonthsAttribute(int months)
```

Comment:
Passing a TimeSpan as an attribute parameter is not possible (only primitive types can be passed as attribute parameters).